src/servers/grpc_server.cc

```diff
@@ -292,6 +292,59 @@ GRPCServer::Create(
   return Status::Success;
 }
 
+Status
+GRPCServer::CreateUniqueEndpointPorts(
+    InferenceServer* server, std::string endpoint_name, uint16_t port,
+    std::unique_ptr<GRPCServer>* grpc_server)
+{
+  // DLIS-162 - provide global defaults and cli overridable options
```

Comment:
Don't include the DLIS since that is our internal tracking and so not useful to GitHub users.